shared/src/common/enums/intance-type/appointment-type.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum AppointmentType {
+  ONLINE = 'online',
+  OFFLINE = 'offline'
+}
```

Comment:
Move such interfaces/enums to the specific entity folder instead of `instance-type`
Example:
```
shared/interfaces/appointment/appointment.interfaces.ts
shared/enums/appointment/appointment-type.enum.ts
```